javascript/javascript2/week3/function-variable.js

```diff
@@ -0,0 +1,39 @@
+/*Create an array with 3 items. All items should be functions. 
+Iterate through the array and call all the functions*/
+
+const array = [];
+
+const firstFunction = () => "this is first function";
+const secondFunction = () => "this is second function";
+const thirdFunction = () => "this is third function";
+
+function pushToArray(element) {
+  array.push(element);
+}
```

Comment:
I would suggest to initialise the array with the elements inside instead of pushing. It may not look that much now, but having to push each element inside the array is actually very time consuming algorithm.